src/pages/Home/Home.css

```diff
@@ -17,6 +17,18 @@
     margins, or anything you like, as long as
     the outcome reflects the Figma. */
 
+#line1{
+    position: absolute;
+    margin: 250px 0 0 150px;
```

Comment:
This is a really good example on why exact measurements fall short to relative measurements. If you zoom in and out on the home page (the root route), you can see that the text doesn't stay centered. Best to use flexbox or css grid for centering, and add margin as needed for an offset.